### ISSUE TITLE
Setting `render_mode` for wrapped Envs not working

### ISSUE THREAD

**Issue by maxmunzel** [ALRhub/fancy_gym#78]
Hi!

I can't figure out how to set the render mode in MP environments. 
The gymnasium docs [say](https://gymnasium.farama.org/api/env/#gymnasium.Env.render) that beginning with 0.25.0, the 
render mode has to be passed to `gym.make()`, not to `env.render()`. But this does not work for some fancy_gym envs, e.g.:

```python
casparfriedrich@fedora ~/t/fancy_gym (master)> python
Python 3.11.6 (main, Oct  3 2023, 00:00:00) [GCC 13.2.1 20230728 (Red Hat 13.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gymnasium
>>> import fancy_gym
>>> env = gymnasium.make("CartPole-v1", render_mode="human")
>>> env = gymnasium.make("fancy_ProMP/HoleReacher-v0", render_mode="human")
/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/envs/registration.py:788: UserWarning: WARN: The environment is being initialised with render_mode='human' that is not in t
he possible render_modes ([]).
  logger.warn(
Traceback (most recent call last):
  File "/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/envs/registration.py", line 802, in make
    env = env_creator(**env_spec_kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: HoleReacherEnv.__init__() got an unexpected keyword argument 'render_mode'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/envs/registration.py", line 802, in make
    env = env_creator(**env_spec_kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/casparfriedrich/thesis/fancy_gym/fancy_gym/envs/registry.py", line 281, in bb_env_constructor
    raw_underlying_env = gym_make(underlying_id, **kwargs)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/envs/registration.py", line 814, in make
    raise type(e)(
TypeError: HoleReacherEnv.__init__() got an unexpected keyword argument 'render_mode' was raised from the environment creator for fancy/HoleReacher-v0 with kwargs ({'n_links': 5, 'random_star
t': True, 'allow_self_collision': False, 'allow_wall_collision': False, 'hole_width': None, 'hole_depth': 1, 'hole_x': None, 'collision_penalty': 100, 'render_mode': 'human'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/envs/registration.py", line 814, in make
    raise type(e)(
TypeError: HoleReacherEnv.__init__() got an unexpected keyword argument 'render_mode' was raised from the environment creator for fancy/HoleReacher-v0 with kwargs ({'n_links': 5, 'random_star
t': True, 'allow_self_collision': False, 'allow_wall_collision': False, 'hole_width': None, 'hole_depth': 1, 'hole_x': None, 'collision_penalty': 100, 'render_mode': 'human'}) was raised from
 the environment creator for fancy_ProMP/HoleReacher-v0 with kwargs ({'underlying_id': 'fancy/HoleReacher-v0', 'mp_wrapper': <class 'fancy_gym.envs.classic_control.hole_reacher.mp_wrapper.MPW
rapper'>, 'mp_type': 'ProMP', '_mp_config_override_register': {}, 'render_mode': 'human'})
```

The way `fancy_gym/examples/examles_motion_primitives.py` tries to set the render mode also does not work It crashes if you set `render = True` in its main.

The following example is based on https://github.com/ALRhub/fancy_gym/blob/c985f2c415decd3c2d9506e40c4f254fe5bf6ec3/fancy_gym/examples/examples_movement_primitives.py#L35
```python
>>> env = gymnasium.make("fancy_ProMP/HoleReacher-v0")
/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/core.py:311: UserWarning: WARN: env.dt to get variables from other wrappers is deprecated and will be removed in v1.0, to g
et this variable you can do `env.unwrapped.dt` for environment variables or `env.get_wrapper_attr('dt')` that will search the reminding wrappers.
  logger.warn(
WARNING:root:Zero Padding ProMP is being used. Only the traj position and velocity can be computed correctly. The other entities are not guaranteed.
/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/core.py:311: UserWarning: WARN: env.random_start to get variables from other wrappers is deprecated and will be removed in 
v1.0, to get this variable you can do `env.unwrapped.random_start` for environment variables or `env.get_wrapper_attr('random_start')` that will search the reminding wrappers.
  logger.warn(
/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/core.py:311: UserWarning: WARN: env.n_links to get variables from other wrappers is deprecated and will be removed in v1.0,
 to get this variable you can do `env.unwrapped.n_links` for environment variables or `env.get_wrapper_attr('n_links')` that will search the reminding wrappers.
  logger.warn(
/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/core.py:311: UserWarning: WARN: env.initial_width to get variables from other wrappers is deprecated and will be removed in
 v1.0, to get this variable you can do `env.unwrapped.initial_width` for environment variables or `env.get_wrapper_attr('initial_width')` that will search the reminding wrappers.
  logger.warn(
>>> env.reset()
(array([ 0.02895356,  1.        ,  1.        ,  1.        ,  1.        ,
        0.99958074,  0.        ,  0.        ,  0.        ,  0.        ,
        0.        ,  0.        ,  0.        ,  0.        ,  0.        ,
        0.46527618, -0.53366953,  5.997904  ], dtype=float32), {})
>>> env.render(mode="human")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/wrappers/order_enforcing.py", line 70, in render
    return self.env.render(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/casparfriedrich/.local/lib/python3.11/site-packages/gymnasium/wrappers/env_checker.py", line 65, in render
    return env_render_passive_checker(self.env, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: env_render_passive_checker() got an unexpected keyword argument 'mode'
```

Am I doing something wrong or is this broken? In the latter case: Is there another way to render the simulation?

**Comment by maxmunzel** [comment]
For reference, these are the library versions in my python env:
[freeze.txt](https://github.com/ALRhub/fancy_gym/files/13044783/freeze.txt)

**Comment by D-o-d-o-x** [comment]
This is an issue with `fancy_gym`.
I do not know of a workaround, but should be fixed soon...

**Comment by D-o-d-o-x** [comment]
Issue is now fixed on master (was addressed by PR #83).
